<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgramShow</class>
 <widget class="QWidget" name="ProgramShow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_Btn" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>450</y>
     <width>800</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(145, 145, 145);</string>
   </property>
   <widget class="QPushButton" name="Btn_AddOrder">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>添加指令</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_Edit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>编辑</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_Debug">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>调试</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_AlterPos">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>修改位置</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_HideDeclare">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>隐藏声明</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_Btn_top" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>800</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(145, 145, 145);</string>
   </property>
   <widget class="QPushButton" name="Btn_MissionAndPro">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>266</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>任务与程序</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_Module">
    <property name="geometry">
     <rect>
      <x>267</x>
      <y>0</y>
      <width>266</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>模块</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_RoutinePro">
    <property name="geometry">
     <rect>
      <x>533</x>
      <y>0</y>
      <width>266</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>例行程序</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_Pos">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>5</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>81</y>
     <width>320</width>
     <height>369</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QStackedWidget {
	border:1px solid gray;
}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="addOrder">
    <widget class="QStackedWidget" name="stackedWidget_addorder">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>320</width>
       <height>319</height>
      </rect>
     </property>
     <widget class="QWidget" name="Common">
      <widget class="QPushButton" name="btn_MoveC">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>2</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>MoveC</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_MoveL">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>47</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>MoveL</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_IF">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>92</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>IF</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_Wait">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>137</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Wait</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_WHILE">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>182</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>WHILE</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_8">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>227</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_9">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>182</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_FOR">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>92</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>FOR</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_11">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>227</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_Set">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>137</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_MoveJ">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>2</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>MoveJ</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_Reset">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>47</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>320</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(145, 145, 145);</string>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>320</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>addorder</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="edit">
    <widget class="QWidget" name="widget_edit" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>320</width>
       <height>370</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
}</string>
     </property>
     <widget class="QPushButton" name="btn_copy">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>复制</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_paste">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>粘贴</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_cut">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>剪切</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="debug">
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>qdebug</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_ShowPro">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>750</width>
     <height>370</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:12pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>50</width>
     <height>370</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:12pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p align=&quot;right&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <zorder>widget_Btn</zorder>
  <zorder>widget_Btn_top</zorder>
  <zorder>label_Pos</zorder>
  <zorder>textBrowser_ShowPro</zorder>
  <zorder>textBrowser_line</zorder>
  <zorder>stackedWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
